com/pythoncom.py

```diff
@@ -1,4 +1,29 @@
 # Magic utility that "redirects" to pythoncomXX.dll
+from typing import Any
+
 import pywintypes
 
 pywintypes.__import_pywin32_system_module__("pythoncom", globals())
+
+
+# This module dynamically re-exports from a C-Extension.
+# `__getattr__() -> Any` prevents checkers attribute access errors
+# without the use of external type stubs.
+# So keep it even if pythoncom.frozen support is removed.
+def __getattr__(name: str) -> Any:
```

Comment:
This incidentally has the exact same benefits as https://github.com/mhammond/pywin32/pull/2418/files#diff-417d6662c94f9f97ab29ec3611822c62e6da513054d38f6dfdc0155fc2f4463b